docs/setup-react-native.md

```diff
@@ -81,3 +81,23 @@ try {
   console.log(error)
 }
 ```
+
+## Using BBS Signatures
+
+When using AFJ inside the React Native environment, temporarily, a dependency for creating keys, sigining and verifying
+with bbs keys must be swapped. Inside your package.json the following must be added:
+
+```diff
++   "resolutions": {
++     "@mattrglobal/bbs-signatures": "react-native-bbs-signatures@0.1.0",
++   },
+    "dependencies": {
+      ...
++     "react-native-bbs-signatures": "0.1.0",
```

Comment:
Can we add `react-native-bbs-signatures` as a peer dependency to the RN wrapper?